Improve install docs (#1760)
plone/documentation: docs/admin-guide/run-plone.md

## Run Plone in foreground mode

Running Plone in foreground mode will show output in the terminal.

> This is recommended while developing a Plone site. The command to be used, depends on the used install method.

Reviewer: ```suggestion
This is recommended while developing a Plone site. The command to be used depends on the install method you used.
```

Reviewer: @stevepiercy help with wording please...my suggestion is also awkward

> This is recommended while developing a Plone site. The command to be used depends on the install method you used.

Reviewer 2: ```suggestion
This is recommended while developing a Plone site.
The command you use depends on the installation method you used.
```
